src/css/main.css

```diff
@@ -120,7 +120,7 @@ section {
 
   .container {
     max-width: 1280px;
-    padding-left: 128px;
-    padding-right: 128px;
+    padding-left: 64px;
```

Comment:
in case of these, i think we should add additional paddings everywhere where it was already used on the desktop screens.